components/de_amp/browser/de_amp_url_loader.cc

```diff
@@ -17,7 +17,9 @@ namespace de_amp {
 
 namespace {
 
-constexpr uint32_t kReadBufferSize = 65536;
+constexpr uint32_t kReadBufferSizeBytes = 65536;
+constexpr uint32_t kMaxBytesToCheck =
+    kReadBufferSizeBytes * 3;  // Should always be a multiple
```

Comment:
Given that we're reading in chunks, we might end up in a case where we read more bytes than `kMaxBytesToCheck` if it is not a multiple of `kReadBufferSizeBytes`, which doesn't break anything, just means we end up reading more bytes than necessary. If we read less than the chunk (which can happen), then this can happen as well.

Reply:
Yes, it's clear. `Should always be a multiple` looks confusing to me, because it seems a referrer to reading a fixed size chunks (that isn't always true)

Reply:
Fair.